content/en/observability_pipelines/use_cases.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Use Cases
+disable_toc: false
+aliases:
+- /path-to-old-doc/
+further_reading:
+- link: "observability_pipelines/set_up_pipelines#set-up-a-pipeline"
+  tag: "Documentation"
+  text: "Set up pipelines"
+---
+
+## Overview
+
+When you create a pipeline in the Observability Pipelines UI, select one of the out-the-box templates to build and deploy pipelines based on your use case.
+
+{{< img src="observability_pipelines/templates_20241003.png" alt="The Observability Pipelines UI showing the seven templates" style="width:100%;" >}}
+
+## Templates
+
+The templates are built for the following use cases:
+
+### Archive Logs
+
+Use the Archive Logs template to store logs in a cloud storage solution (Amazon S3, Google Cloud Storage, or Azure Storage). The archived logs are stored in a Datadog-rehydratable format, so that they can be rehydrated in Datadog as needed. This is useful when:
```

Comment:
```suggestion
Use the Archive Logs template to store logs in a cloud storage solution (Amazon S3, Google Cloud Storage, or Azure Storage). The archived logs are stored in a Datadog-rehydratable format so that they can be rehydrated in Datadog as needed. This is useful when:
```